suite-native/module-home/src/components/PortfolioGraphHeader.tsx

```diff
@@ -39,7 +39,13 @@ const Balance = () => {
     const point = useAtomValue(selectedPointAtom);
 
     return (
-        <FiatAmountFormatter value={String(point.value)} variant="titleLarge" color="textDefault" />
+        <FiatAmountFormatter
+            value={String(`${point.value}`)}
```

Comment:
Why it is wrapped both in upper ticks and String? String itself should be enough.

Reply:
fixed